### PR TITLE
[onert] Introduce int8 and int16 type

### PR DESCRIPTION
**Pull request by hseok-oh**
Introduce data type
- QUANT_INT8_ASYMM: int8 quantization operation input/output tensor data type (int8 weight tensor uses QUANT_INT8_SYMM)
- QUANT_INT16_ASYMM: int16 quantization operation input/output tensor data type

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

--- 

Related issue: #4664

### CODE REVIEW

File: runtime/onert/core/include/ir/DataType.h [hunk 1]
```diff
@@ -35,6 +35,8 @@ enum class DataType
   QUANT_INT8_SYMM = 6,
   FLOAT16 = 7,
   INT64 = 8,
+  QUANT_INT8_ASYMM = 9,
+  QUANT_INT16_ASYMM = 10,
```
Comment (glistening) [inline runtime/onert/core/include/ir/DataType.h:39]: @hseok-oh Could you please handle this types in `sizeOfDataType(DataType data_type)` also?

Comment (hseok-oh) [reply runtime/onert/core/include/ir/DataType.h:39]: @glistening I've updated. PTAL

**Review by glistening — APPROVED** [review]
LGTM

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**